aesthetic_ants/penguin/main.py

```diff
@@ -1,11 +1,27 @@
 import pyglet
+from enemy import Enemy
```

Comment:
This import needs to be fixed to the following as relative imports must be explicit. It doesn't work when I'm testing the PR on terminal for some reason otherwise.
```suggestion
from .enemy import Enemy
```